web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -151,24 +152,23 @@ function renderVersionStatus(version, app, match, viewLogs) {
   }
 
   if (!isPastVersion && !isPendingDeployedVersion) {
-    const deployedAndSkippedChecks = version.preflightSkipped ? " (checks skipped)" : "";
     return (
       <div className="flex alignItems--center">
         <div className="flex alignItems--center">
           <div
             data-tip={`${version.versionLabel || version.title}-${version.sequence}`}
             data-for={`${version.versionLabel || version.title}-${version.sequence}`}
             className={classNames("icon", {
-              "checkmark-icon": version.status === "deployed" || version.status === "merged" || version.status === "pending",
-              "exclamationMark--icon": version.status === "opened" || (version.preflightSkipped && version.status !== "deployed"),
+              "checkmark-icon": showCheckmarkIcon,
+              "exclamationMark--icon": version.status === "opened" || (version.preflightSkipped && !showCheckmarkIcon),
               "grayCircleMinus--icon": version.status === "closed",
               "error-small": version.status === "failed" || preflightsFailed
             })}
           />
           {version.status === "deploying" && <Loader className="flex alignItems--center" size="20" />}
           <span className={classNames("u-fontSize--small u-fontWeight--medium u-lineHeight--normal u-marginLeft--5", {
             "u-textColor--accent": version.status === "deployed" || version.status === "merged",
-            "u-textColor--warning": version.status === "opened" || (version.preflightSkipped && version.status !== "deployed"),
+            "u-textColor--warning": version.status === "opened",
             "u-textColor--bodyCopy": version.status === "closed" || version.status === "pending" || version.status === "pending_preflight",
             "u-textColor--error": version.status === "failed" || preflightsFailed
```

Comment:
why is this different from this https://github.com/replicatedhq/kots/pull/1995/files#diff-a5ee13382325550db7f0c09495c03715d7dbb7c581dfce6d03a32d0ee81360cdR204